tests/conftest.py

```diff
@@ -300,3 +301,46 @@ def identity_simple():
     i.provides.add(UserNeed(1))
     i.provides.add(Need(method='system_role', value='any_user'))
     return i
+
+
+@pytest.fixture(scope="session")
+def create_record():
```

Comment:
Minor: is it possible to somehow use one of the existing minimal records to test with to avoid having too many record test fixtures? Also, to create the record, I think it's better if the service is being used ``service.create(...)``.